### PR TITLE
Added support for tile margin and spacing

### PR DESCRIPTION
**Pull request by diegomgarcia**

### CODE REVIEW

File: lib/src/tile.dart [hunk 1]
```diff
@@ -29,8 +31,10 @@ class Tile {
   }
 
   Tile(this.tileId, this.tileset) {
-    width = tileset.width;
+    width = tileset.width ;
```
Comment (luanpotter) [inline lib/src/tile.dart:34]: please remove this space

File: lib/src/tile.dart [hunk 2]
```diff
@@ -10,6 +10,8 @@ class Tile {
 
   int width;
   int height;
+  int spacing;
+  int margin;
 
   Map<String, String> properties = new Map();
```
Comment (luanpotter) [inline lib/src/tile.dart:16]:
```suggestion
  Map<String, String> properties = {};
```

Comment (diegomgarcia) [reply lib/src/tile.dart:16]: Hi that one wasn't from my commit.

File: lib/src/tile.dart (hunk 1 above)
Comment (diegomgarcia) [reply lib/src/tile.dart:34]: ops didn't saw that, thanks.

**Pull request merged**